internal/handlers/web/assets/app.js

```diff
@@ -89,17 +89,17 @@ document.addEventListener("DOMContentLoaded", function () {
 
   // Function to fetch and display the application version
   function fetchAppVersion() {
-    fetch('/version').then(response => {
-      if (!response.ok) {
-        throw new Error('Failed to fetch app version');
-      }
-      return response.text();
-    })
-      .then(response => response.text())
-      .then(version => {
-        document.getElementById('appVersion').textContent = version;
+    fetch("/version")
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error("Failed to fetch app version");
+        }
+        return response.text();
       })
-      .catch(error => console.error('Error fetching app version:', error));
+      .then((version) => {
+        document.getElementById("appVersion").textContent = version;
```

Comment:
Consider handling the case where `document.getElementById('appVersion')` returns null. This could happen if the element with id 'appVersion' does not exist in the DOM. This will prevent potential errors when trying to set `textContent` on a null object. [important]